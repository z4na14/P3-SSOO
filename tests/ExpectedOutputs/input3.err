[ERROR][factory_manager] Invalid file.
