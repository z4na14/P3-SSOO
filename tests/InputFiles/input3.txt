4 5 10 2 1 2 3 13 5 2