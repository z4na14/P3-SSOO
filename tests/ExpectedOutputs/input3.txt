[OK][factory_manager] Process_manager with id 5 has been created.
[OK][factory_manager] Process_manager with id 1 has been created.
[OK][factory_manager] Process_manager with id 13 has been created.
[OK][process_manager] Process_manager with id 5 waiting to produce 2 elements.
[OK][process_manager] Process_manager with id 1 waiting to produce 3 elements.
[OK][process_manager] Process_manager with id 13 waiting to produce 2 elements.
[OK][process_manager] Belt with id 5 has been created with a maximum of 10 elements.
[OK][process_manager] Belt with id 1 has been created with a maximum of 2 elements.
[OK][process_manager] Belt with id 13 has been created with a maximum of 5 elements.
[OK][queue] Introduced element with id 0 in belt 5.
[OK][queue] Introduced element with id 0 in belt 13.
[OK][queue] Introduced element with id 1 in belt 5.
[OK][queue] Introduced element with id 1 in belt 13.
[OK][queue] Obtained element with id 0 in belt 13.
[OK][queue] Obtained element with id 1 in belt 13.
[OK][queue] Introduced element with id 0 in belt 1.
[OK][queue] Introduced element with id 1 in belt 1.
[OK][queue] Obtained element with id 0 in belt 1.
[OK][queue] Obtained element with id 1 in belt 1.
[OK][queue] Introduced element with id 2 in belt 1.
[OK][queue] Obtained element with id 2 in belt 1.
[OK][process_manager] Process_manager with id 13 has produced 2 elements.
[OK][queue] Obtained element with id 0 in belt 5.
[OK][queue] Obtained element with id 1 in belt 5.
[OK][process_manager] Process_manager with id 1 has produced 3 elements.
[OK][process_manager] Process_manager with id 5 has produced 2 elements.
[OK][factory_manager] Process_manager with id 5 has finished.
[OK][factory_manager] Process_manager with id 1 has finished.
[OK][factory_manager] Process_manager with id 13 has finished.
[OK][factory_manager] Finishing.
